hydrolib/core/io/ini/util.py

```diff
@@ -81,12 +87,16 @@ def validate_correct_length(cls, values):
 
         for field_name in field_names:
             field = values.get(field_name)
-            if field is not None and not (
-                isinstance(field, list) and len(field) == length
-            ):
+            if field is not None:
+                if not (isinstance(field, list) and len(field) == length):
```

Comment:
I know this check was already there, but now that I think about it, the `if not (isinstance(field, list)` should not really be necessary right? If someone adds fields to this function that are not lists, it is a programming mistake not a valid situation. Or am I missing something? If it is an invalid function input I would remove this first check.

Reply:
fair point, changed.